src/themes/OLH/templates/core/nav.html

```diff
@@ -26,15 +26,15 @@
         {% if journal_settings.general.enable_editorial_display %}
             {% if journal_settings.general.multi_page_editorial %}
             <li class=".is-dropdown-submenu-parent">
-                <a href="#" data-options="is_hover:true; hover_timeout:5000">{% trans 'Editorial Team' %}</a>
+                <a href="#" data-options="is_hover:true; hover_timeout:5000">{{ journal_settings.styling.editorial_group_page_name }}</a>
```

Comment:
We only provide a default for the English language on "journal_defaults.json". As a result, I think any Janeway installs running a different language for the front-end would lose this translation. We could solve this with a data migration that loads in the existing translations in the PO files onto the database for this value.